lib/stages/visRawReader.hpp

```diff
@@ -192,4 +208,61 @@ class visRawReader : public kotekan::Stage {
     double sleep_time;
 };
 
+class ensureOrdered : public kotekan::Stage {
```

Comment:
This turned out not to be used in this release because we are instead relying on the fact that the network buffers receive frames in the same order they are sent. I'm not sure what is the best thing to do with this code since it might still be useful in the future.